src/suites/cts/validation/createPipelineLayout.spec.ts

```diff
@@ -0,0 +1,70 @@
+export const description = `
+createPipelineLayout validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+function clone(descriptor: GPUBindGroupLayoutDescriptor): GPUBindGroupLayoutDescriptor {
+  return JSON.parse(JSON.stringify(descriptor));
+}
+
+export const g = new TestGroup(ValidationTest);
+
```

Comment:
Can we also have a test 4 BGLs being a success and 5BGLs being an error?

Reply:
Added this one and several others as TODOs. @beaufortfrancois please feel free to take those on.